projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/cutsomer-ticketing.ts

```diff
@@ -0,0 +1,38 @@
+import { standardUser } from '../../sample-data/shared-users';
+import { generateMail, randomString } from '../user';
+import { clickHamburger } from '../checkout-flow';
+
+export function loginRegisteredUser() {
+  standardUser.registrationData.email = generateMail(randomString(), true);
+  cy.requireLoggedIn(standardUser);
+  cy.reload();
+}
+
+export function openMyAccountDropDown(){
```

Comment:
How about removing the type of element rendering to something like openMyAccountMenu?
Since mobile is a hamburger that doesn't look like a DropDown

Reply:
Agree. Renamed to `clickMyAccountMenuOption`.